src/java.base/share/classes/java/util/Properties.java

```diff
@@ -814,10 +821,16 @@ public void save(OutputStream out, String comments)  {
      * character {@code !} then an ASCII {@code #} is written out
      * after that line separator.
      * <p>
-     * Next, a comment line is always written, consisting of an ASCII
-     * {@code #} character, the current date and time (as if produced
-     * by the {@code toString} method of {@code Date} for the
-     * current time), and a line separator as generated by the {@code Writer}.
+     * If the {@systemProperty java.util.Properties.storeDate} is set and
+     * is non-empty (as determined by {@link String#isEmpty()  String.isEmpty}),
+     * a comment line is written as follows.
+     * First, a {@code #} character is written, followed by the contents
+     * of the property, followed by a line separator.
```

Comment:
Maybe you should refer to how the comment is written out above, since you use the same method. The spec changes as written above does not specify how newlines in the property would be handled (which is possible to get, I believe, even if it means an intricate command line escape dance)

Reply:
Hello Magnus, 
> Maybe you should refer to how the comment is written out above, since you use the same method. 

I've updated the javadoc to capture this part of the detail.
> The spec changes as written above does not specify how newlines in the property would be handled (which is possible to get, I believe, even if it means an intricate command line escape dance)

The new tests that were added in this PR has one specific test to verify how line terminators are dealt with, if the system property value has them https://github.com/openjdk/jdk/pull/5372/files#diff-220f7bcc6d1a7ec33764f81eb95ccb0f69444e1eb923b015025e2140c3ffe145R280